.test-infra/jenkins/job_PostCommit_Java_Nexmark_Flink.groovy

```diff
@@ -43,6 +43,8 @@ NoPhraseTriggeringPostCommitBuilder.postCommitJob('beam_PostCommit_Java_Nexmark_
       switches('-Pnexmark.runner=":runners:flink:1.9"' +
               ' -Pnexmark.args="' +
               [NexmarkBigqueryProperties.nexmarkBigQueryArgs,
+              '--runner=FlinkRunner',
+              '--shutdownSourcesOnFinalWatermark=true',
```

Comment:
It looks like we still want to use `TestFlinkRunner` here because it auto-sets the `--shutdownSourcesOnFinalWatermark` flag.

Reply:
Wouldn't it be enough to just do the [above change](https://github.com/apache/beam/pull/10664/files#diff-3bbf75b566341b4b08d71632cd27e022R32)?

Reply:
No, if we use `TestFlinkRunner` the results will be saved in the table with the corrupted data (the runs with the `DirectRunner` results. If we use FlinkRunner we will get the data in the same table with the historical data previous to 2019/01.